config/config.yaml

```diff
@@ -0,0 +1,27 @@
+serverPort: 8080
+
+indexerConfig:
+  maxMessagesPerChunk: 100
+  maxBytesPerChunk: 100000
+  commitDurationSecs: 10
+  refreshDurationSecs: 11
+  staleDurationSecs: 7200
+  dataTransformer: "api_log"
+  dataDirectory: "/tmp"
+
+kafkaConfig:
+  kafkaTopic: ${KAFKA_TOPIC:-test-topic}
+  kafkaTopicPartition: "0"
```

Comment:
Can we make kafkaTopic/kafkaTopicPartition/kafkaBootStrapServers/kafkaClientGroup/s3Bucket/dataTransformer

read an ENV var as well with the default that works best for local debugging ?